### ISSUE TITLE
Dropdown: Show Clear Keyboard Binding

### ISSUE THREAD

**Issue by pyramid-johng** [primefaces/primevue#3834]
### Describe the feature you would like to see added

The `Dropdown` with the `showClear` option `true` should have a keyboard bindng for the "clear" command.  Propose the `Delete` key.

### Is your feature request related to a problem?

The `Dropdown` does not allow me to click on the "X" icon under certain circumstances, and requires lots of input focus toggling to make it respond.

### Describe the solution you'd like

In any state, the user should be able to use the "clear" keyboard binding to clear the selection.

### Additional context

Every action supported via mouse should also be supported via keyboard, so this is an accessibility enhancement.

**Comment by SamuelWei** [comment]
@mertsincan This is missing in v4
